mempool/src/shared_mempool/network.rs

```diff
@@ -444,11 +476,21 @@ impl<NetworkClient: NetworkClientInterface<MempoolSyncMsg>> MempoolNetworkInterf
         &self,
         peer: PeerNetworkId,
         batch_id: MultiBatchId,
-        transactions: Vec<SignedTransaction>,
+        // For each transaction, we include the ready time in millis since epoch
+        transactions: Vec<(SignedTransaction, u64)>,
+        use_mempool_sync_message_v2: bool,
```

Comment:
Do we need to pass this? We should be able to read from `self.mempool_config`

Reply:
Done.